src/dotnet/APIView/APIViewTest/ParameterTests.cs

```diff
@@ -25,11 +25,12 @@ public void ParameterTestNoRefKindStringDefaultValue()
             }
 
             Assert.False(param == null || num == null);
-            Assert.Equal("T", param.Type);
+            Assert.Single(param.Type.Tokens);
+            Assert.Equal("T", param.Type.Tokens[0].DisplayString);
+            Assert.Equal(TypeReference.TypeName.SpecialType, param.Type.Tokens[0].Type);
```

Comment:
I would just call it, TypeArgument instead of SpecialType if that's what it represents.